tests/files/nginx_config_client_max_body_size.txt

```diff
@@ -28,6 +28,6 @@ server {
         proxy_ignore_headers Cache-Control Expires;
     }
 
-    access_log /dev/stdout content_cache;
```

Comment:
I believe this will prevent the logs to be viewed with "kubectl logs", for example. And will be kept inside the container.

Reply:
@gtrkiller , since you can't tail the logs printed to stdout unless you mount the file they are written to on the host maybe it would be possible to keep this configuration and `tail` this log so that it is still printed to stdout. Have you considered this approach?

Reply:
The approach I ended utilizing is to make nginx log in two files, one for stdout/stderr and one for access/error.log. You can look at the currently changed files. Does that solve the problem?

Reply:
Or something like ingress does here:
https://github.com/kubernetes/ingress-nginx/blob/main/rootfs/Dockerfile#L82

Reply:
It looks like @amandahla 's solution will imply less IO

Reply:
AFAIK if we do that, we wouldn't be able to see the access/error.log files (have researched about this before), and I need a file containing exclusively logs I can print from to process the action.

Reply:
Have you tried logging stdout and creating a softlink? The other way around (loging to file and creating the link to stdout) won't probably work

Reply:
The `/dev/stdout` is write-only, so read the file that's softlinked to `/dev/stdout` is not possible.

There's another solution that's use the Kubernetes in cluster config to read the pod's 
```
import kubernetes

kubernetes.config.load_incluster_config()
core_v1_api = kubernetes.client.CoreV1Api()
core_v1_api.read_namespaced_pod_log(
    name=self.unit.name.replace("/", "-"),
    namespace=self.model.name, 
    container=<container-name>
)
```

Reply:
The thing with reading the pod's log directly is that if the pod happpens to contain dates with this format (+[%d/+%b/%Y:%H:%M:%S) it will mess up my metrics for the action, that is why I preferred to write the logs to access/error.log exclusively in the first place

Reply:
There's a `since_seconds` argument in `read_namespaced_pod_log` that query log records in a certain period of time, does this helps?

Reply:
I can already filter that with the awk command in fact, so it helps but it is already covered

Reply:
In this case, should we think about add storage in metadata.yaml?

https://juju.is/docs/sdk/storage
https://github.com/jnsgruk/zinc-k8s-operator/blob/main/metadata.yaml

Reply:
Would there also be any impact in case multiple content caches are deployed for the same application? I'm not familiar with this charm so not sure what the usual deployment of it would be